--- lapacke.h	2016-08-31 20:58:42.000000000 -0700
+++ lapacke.h.new	2017-06-27 16:00:38.661811608 -0700
@@ -12521,8 +12521,10 @@
 #define LAPACK_dsysv_rook LAPACK_GLOBAL(dsysv_rook,DSYSV_ROOK)
 #define LAPACK_csysv_rook LAPACK_GLOBAL(csysv_rook,CSYSV_ROOK)
 #define LAPACK_zsysv_rook LAPACK_GLOBAL(zsysv_rook,ZSYSV_ROOK)
+/*
 #define LAPACK_csyr LAPACK_GLOBAL(csyr,CSYR)
 #define LAPACK_zsyr LAPACK_GLOBAL(zsyr,ZSYR)
+*/
 #define LAPACK_ilaver LAPACK_GLOBAL(ilaver,ILAVER)
 // LAPACK 3.6.0
 #define LAPACK_sggsvd3 LAPACK_GLOBAL(sggsvd3,SGGSVD3)
